src/Uno.UI/UI/Xaml/UIElement.RoutedEvents.cs

```diff
@@ -434,6 +438,22 @@ private static object CoerceRoutedEventFlag(DependencyObject dependencyObject, o
 			remove => RemoveHandler(DropCompletedEvent, value);
 		}
 
+#if __WASM__
+		public event KeyEventHandler PreviewKeyDown
+		{
+			add => AddHandler(PreviewKeyDownEvent, value, false);
+			remove => RemoveHandler(PreviewKeyDownEvent, value);
+		}
+#endif
+
+#if __WASM__
+		public event KeyEventHandler PreviewKeyUp
+		{
+			add => AddHandler(PreviewKeyUpEvent, value, false);
+			remove => RemoveHandler(PreviewKeyUpEvent, value);
+		}
+#endif
```

Comment:
This could be a single `#if __WASM__` rathern than two

Reply:
Thank you, @Youssef1313